.github/workflows/lint_test.yml

```diff
@@ -0,0 +1,28 @@
+name: Linting and testing
+
+on:
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
use `npm ci` as it's faster and more stable in terms of versions installed than `npm i`